packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -0,0 +1,112 @@
+import { Component, createElement } from 'react';
+import {
+    bool,
+    func,
+    shape,
+    number,
+    objectOf,
+    arrayOf,
+    string
+} from 'prop-types';
+import { Price } from '@magento/peregrine';
+
+import classify from 'src/classify';
+import Carousel from 'src/components/ProductImageCarousel';
+import Quantity from 'src/components/ProductQuantity';
+import RichText from 'src/components/RichText';
+import defaultClasses from './productFullDetail.css';
+
+class ProductFullDetail extends Component {
+    static propTypes = {
+        classes: objectOf(string).isRequired,
```

Comment:
These classes should be documented. See comment from @jimbo https://github.com/magento-research/pwa-studio/pull/90#discussion_r199540398